crates/ruff_db/src/files.rs

```diff
@@ -159,9 +161,11 @@ impl Files {
         tracing::trace!("Adding virtual file {}", path);
         let virtual_file = VirtualFile(
             File::builder(FilePath::SystemVirtual(path.to_path_buf()))
+                .path_durability(Durability::HIGH)
                 .status(FileStatus::Exists)
                 .revision(FileRevision::zero())
                 .permissions(None)
+                .permissions_durability(Durability::HIGH)
```

Comment:
What happens if permissions on a file do change? Do we treat them as immutable and ignore such changes? Do we not even watch for permission changes? I assume permissions aren't part of the file identity...

Reply:
Virtual files don't have a permission, it's always set to `None`